Source/OxyPlot/Axes/DateTimeAxis.cs

```diff
@@ -1073,6 +1076,7 @@ private DateTime ConvertToLocalTime(double x)
             }
             else if (range.TotalDays > 1)
             {
+                minorStepInDays = true;
                 var nextNiceDay = startTime.NextNiceDay(niceDayNumbers);
```

Comment:
This seems risky - maybe add a condition `minorStepInDays = range.TotalDays > 5`?